ranger/config/rifle.conf

```diff
@@ -129,6 +130,8 @@ mime ^video,       has mplayer,  X, flag f = mplayer -fs -- "$@"
 mime ^video|audio, has vlc,      X, flag f = vlc -- "$@"
 mime ^video|audio, has totem,    X, flag f = totem -- "$@"
 mime ^video|audio, has totem,    X, flag f = totem --fullscreen -- "$@"
+mime ^audio|ogg$,  has audacity, X, flag f = audacity -- "$@"
```

Comment:
Do we need the explicit `ogg`?

Reply:
Probably not. Perhaps this was to include ogx (mime application/ogg instead of audio/ogg)?  I would say better to remove it though and just use `^audio` there.